src/main/java/utility/FigureSupplier.java

```diff
@@ -0,0 +1,49 @@
+package utility;
+
+import java.util.Random;
+import model.Color;
+import model.Figure;
+import model.figure.Circle;
+import model.figure.IsoscelesTrapezoid;
+import model.figure.Rectangle;
+import model.figure.RightTriangle;
+import model.figure.Square;
+
+public class FigureSupplier {
+    public static final String DEFAULT_COLOR = "white";
+    public static final int DEFAULT_DATA = 10;
+    public static final int MAX_RANDOM_VALUE = 100;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+    private final Color[] colors = Color.values();
+
+    public Figure getRandomFigure() {
```

Comment:
The `getRandomFigure()` method should use a constant for the number of figure types instead of `colors.length` to avoid confusion and ensure that the number of figure types is clear and intentional.